tools/spark-reader-v1/src/main/java/com/vesoft/nebula/reader/NebulaPartition.java

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.reader;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.spark.Partition;
+
+public class NebulaPartition implements Partition {
```

Comment:
`Partition` is the identifier for a partition in the RDD. Why it include the total partition number?

Reply:
totalPartition is used in getScanParts method: 

``` java
public List<Integer> getScanParts(int totalPart) {
    List<Integer> scanParts = new ArrayList<>();
    int currentPart = index + 1;
    while (currentPart <= totalPart) {
        scanParts.add(currentPart);
        currentPart += totalPartition;
    }
    return scanParts;
}
```

it seem more reasonable that remove it from NebulaPartition, and the getScanParts method add totalPartition parameter.